permissions/page.html

```diff
@@ -0,0 +1,24 @@
+<html>
```

Comment:
There are some issues with the HTML here. I'm not sure which of them are nits.
* no DOCTYPE
* no `meta charset` declaration (this one gives an error in the console)
* no `<title>`
* `<div class="container">` is not closed (I think indentation would have made this more obvious)